frontend/src/pages/admin/Template/utils.js

```diff
@@ -17,6 +18,7 @@ const templateFields = [
 const defaultTemplateValues = {
     disableEditor: false,
     httpAdminRoot: '',
+    dashboardUI: '',
```

Comment:
This should reflect the actual default value:
```suggestion
    dashboardUI: '/ui',
```